go/staking/api/sanity_check.go

```diff
@@ -73,21 +73,21 @@ func SanityCheckDelegations(account *Account, delegations map[signature.PublicKe
 	sharesExpected := account.Escrow.Active.TotalShares
 
 	if shares.Cmp(&sharesExpected) != 0 {
-		return fmt.Errorf("staking: sanity check failed: all shares of all delegations for account don't add up to account's total active shares in escrow")
+		return fmt.Errorf("staking: sanity check failed: all shares of all delegations (%s) for account with ID: %s don't add up to account's total active shares in escrow (%s)", shares, id, sharesExpected)
```

Comment:
but the caller will put the ID in their message

also below

Reply:
The thing is that `SanityCheckDelegations()` is also called in:
https://github.com/oasislabs/oasis-core/blob/a2db858c3e7c3edb87c36818837c96a2c96401b3/go/staking/api/sanity_check.go#L191-L200

where it didn't print out the account id or other useful info previously.

(Same for other functions, `SanityCheckDebondingDelegations()` and `SanityCheckAccountShares()`.)

I've updated `go/consensus/tendermint/apps/supplementarysanity/checks.go` to not return duplicate info.

Reply:
oh, thanks